src/modules/ekf2/EKF2.cpp

```diff
@@ -2405,6 +2405,15 @@ void EKF2::UpdateGpsSample(ekf2_timestamps_s &ekf2_timestamps)
 			return; //TODO: change and set to NAN
 		}
 
+		if (fabsf(_param_ekf2_gps_yaw_off.get()) > 0.f) {
+			if (!PX4_ISFINITE(vehicle_gps_position.heading_offset) && PX4_ISFINITE(vehicle_gps_position.heading)) {
+				// Apply offset
+				float raw_yaw_offset = matrix::wrap_pi(math::radians(_param_ekf2_gps_yaw_off.get()));
```

Comment:
This is nitpicky, but in what way is this a raw yaw offset rather than just yaw offset?

Reply:
![image](https://github.com/user-attachments/assets/abb52b05-a861-4825-af1a-d30122826d43)

Reply:
From what I remember I wasn't sure how offset by itself was actually used and if it was adjusted in EKF2 somewhere else down the line. Hence "raw" as in coming right from the param before any other changes.

Reply:
Fixed